packages/components-react/projects/react-wrapper/tests/unit/specs/public-api.spec.ts

```diff
@@ -4,13 +4,15 @@ import {
   skipCheckForPorscheDesignSystemProviderDuringTests,
   skipPorscheDesignSystemCDNRequestsDuringTests,
   useToastManager,
+  IconName,
+  TabsChangeEvent,
 } from '../../../src/public-api';
 
 it('should expose componentsReady()', () => {
   expect(typeof componentsReady).toBe('function');
 });
 
-it('should expose PorscheDesignSystemProvider()', () => {
+it('should expose PorscheDesignSystemProvider', () => {
```

Comment:
If you remove `()` here, I would remove `()` also in other tests in this file to be consistent:

```
it('should expose componentsReady', () => {
```

```
it('should expose skipCheckForPorscheDesignSystemProviderDuringTests', () => {
```

```
it('should expose skipPorscheDesignSystemCDNRequestsDuringTests', () => {
```

Reply:
No, those are functions. The ones were I removed it are either classes or components.